api/v1beta1/ironicconductor_types.go

```diff
@@ -120,6 +120,20 @@ type IronicConductorSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
 	// TLS - Parameters related to the TLS
 	TLS tls.Ca `json:"tls,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=Disabled
+	// +kubebuilder:validation:Enum:=Enabled;Disabled
+	// Whether to enable graphical consoles. NOTE: Setting this to Enabled is not supported.
+	GraphicalConsoles string `json:"graphicalConsoles"`
+
+	// +kubebuilder:validation:Optional
+	// NoVNCProxy - Ironic NoVNCProxy Container Image
+	NoVNCProxy string `json:"novncproxy,omitempty"`
```

Comment:
`s/NoVNCProxy/NoVNCProxyImage/` ? All the other types that set container images have the `Image` at the end.